src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -270,7 +270,7 @@ public static int LastIndexOf(ref byte searchSpace, int searchSpaceLength, ref b
                                 return bitPos + offset;
                             }
                             // Clear the highest set bit.
-                            mask = BitOperations.ResetBit(mask, bitPos);
+                            mask = BitOperations.ToggleBit(mask, bitPos);
```

Comment:
Note: It seems likely this code could be optimized by reversing the bit order to allow use of `ResetLowestSetBit` instead, avoiding the `bitPos` data dependency.